src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -81,7 +85,99 @@ protected bool InitializeFileWatcherIfEnabled()
             return false;
         }
 
-        public abstract Task Invoke(object[] parameters);
+        public Task Invoke(object[] parameters)
```

Comment:
This was a refactoring that I've been meaning to do for a while. We have a lot of duplicated code in each of the invokers (start/end events, etc.) that needed to be centralized :)